django/core/management/commands/makemigrations.py

```diff
@@ -48,6 +48,10 @@ def add_arguments(self, parser):
             '-n', '--name',
             help="Use this name for migration file(s).",
         )
+        parser.add_argument(
+            '--no-header', action='store_true',
```

Comment:
You should could use `action='store_false', dest='include_header'` to have the CLI parsing logic deal with the double negation logic.